### PR TITLE
Redesign TOC & Samples overview: Setup, GetStarts, SampApps, DeplSamps

### PR DESCRIPTION
**Pull request by mikehoffms**
Rendered articles for review:
* Promoted in TOC, but unchanged content: https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/how-to/machine-setup?branch=pr-en-us-2054
* Review the several TOC buckets below that, including their firstchild nav/overview pages.  These 3 landing pages replace the old long detailed page that tried to cover all getstarts & samples: 
   * Getting Started tutorials
   * Sample apps
   * Deployment samples

Instead of a single long page with breadth of the entire samples repo (https://docs.microsoft.com/en-us/microsoft-edge/webview2/code-samples-links), that was hard to follow, the info is now distributed: spread across 3 landing pages.  Previous PRs added self-sufficient info into each article, so don't need details in landing pages.

### CODE REVIEW

**Review by JasonAndrewWriter — APPROVED** [review]

**Comment by JasonAndrewWriter** [comment]
Looks good.

File: microsoft-edge/webview2/code-samples-links.md [hunk 1]
```diff
@@ -1,213 +1,49 @@
...
 ms.prod: microsoft-edge
 ms.technology: webview
-ms.date: 04/27/2022
+ms.date: 07/14/2022
 ---
-# Sample Code for WebView2
+# Sample apps
 
-This article is about understanding what samples, `.sln` files, and `README.md` files are in the [WebView2Samples repo](https://github.com/MicrosoftEdge/WebView2Samples).
+Sample apps that use WebView2 are available in the **WebView2Samples** repo, for various frameworks or platforms.
```
Comment (nishitha-burman) [inline microsoft-edge/webview2/code-samples-links.md:13]: We should add a link to the sample apps repo here: https://github.com/MicrosoftEdge/WebView2Samples/tree/main/SampleApps

File: microsoft-edge/webview2/code-samples-links.md [hunk 2]
```diff
@@ -1,213 +1,49 @@
...
+These articles provide the specific steps to set up your machine and build, run, and update each sample.
 
-1. Open one of the `.sln` files.  For example, open your local copy of the multi-platform solution file [WebView2Samples/SampleApps/WebView2Samples.sln](https://github.com/MicrosoftEdge/WebView2Samples/blob/main/SampleApps/WebView2Samples.sln) (downloaded as path `WebView2Samples-main/SampleApps/WebView2Samples.sln`) in Microsoft Visual Studio.  When you open that solution file in Visual Studio, **Solution Explorer** contains the following WebView2 samples, as projects:
 
-   ![Solution Explorer for the WebView2Samples repo, showing the WebView2 samples as projects.](media/solution-file-webview2samples.png)
+#### Sample apps vs. completed Getting Started tutorials
 
-This particular `.sln` file doesn't include the Get Started projects, which are separate `.sln` files, one per platform.
+For some frameworks or platforms, the samples repo contains both a completed Getting Started tutorial and a sample app.  The sample apps demonstrate more features than the completed Getting Started tutorials.
```
Comment (nishitha-burman) [inline microsoft-edge/webview2/code-samples-links.md:30]: The sample apps are home to sample code in the reference docs and conceptual docs. These apps showcase more of our APIs than then Getting Started tutorials.

File: microsoft-edge/webview2/code-samples-links.md [hunk 3]
```diff
@@ -1,213 +1,49 @@
...
-| Windows Presentation Foundation (WPF) .NET | [WebView2WpfBrowser](https://github.com/MicrosoftEdge/WebView2Samples/tree/main/SampleApps/WebView2WpfBrowser) | Embeds a WebView2 control within a WPF application.<br/><br/>  Built as a WPF project in Visual Studio 2019.  Uses C# and HTML/CSS/JavaScript in the WebView2 environment. |
-| Chrome DevTools Protocol (CDP) in WPF | [WV2CDPExtensionWPFSample](https://github.com/MicrosoftEdge/WebView2Samples/tree/main/SampleApps/WV2CDPExtensionWPFSample) | Uses Chrome [DevTools Protocol](../devtools-protocol-chromium/index.md) functions using a `DevToolsProtocolHelper` object in WebView2.  Demonstrates the usage patterns of the WebView2 CDP extension in WPF.  This application is built with the [WebView2 CDP Extension](https://aka.ms/webviewcdp) that defines all CDP methods, events, and types.<br/><br/>  Built as a WPF project in Visual Studio 2019.  Uses C# in the WebView2 environment. |
-
-#### Deployment samples
+#### Articles, readmes, sample directories, and solution files
```
Comment (nishitha-burman) [inline microsoft-edge/webview2/code-samples-links.md:33]: What is this section trying to convey?

**Review by nishitha-burman — CHANGES_REQUESTED** [review]

File: microsoft-edge/webview2/code-samples-links.md (hunk 3 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/code-samples-links.md:33]: I think this table is dup'ing the TOC and is dup'ing details that are already in the individual sample articles.  There is one point that does belong in this overarching article: the special nature of the Win32 / multi-platform sample/solution (currently called out in Notes column on right side of table).  For maintainability, and simplicity of presentation, we need to eliminate dup'ing of details.  Not sure if this applies to the getstart landing page's table & Deployment landing page's table.

Consider the Readme's Redesign PR in the Samples repo, which simplifies & removes dup'n of details; that's working, and can do same simplifying/streamlining here.

**Comment by oumaidi** [comment]
นี่เป็นช่วงเวลาที่ฉันว่าง:


วันพฤหัสบดีที่ 8 กันยายน ค.ศ. 2022



00:00 - 00:30 (GMT+7)




ทำให้ง่ายขึ้นโดย Outlook<https://aka.ms/Dcupce>




รับ Outlook สำหรับ Android<https://aka.ms/AAb9ysg>

________________________________
จาก: nishitha-burman ***@***.***>
ส่งแล้ว: วันเสาร์ที่ 16 กรกฎาคม ค.ศ. 2022 05:19
ถึง: MicrosoftDocs/edge-developer ***@***.***>
สำเนาถึง: Subscribed ***@***.***>
ชื่อเรื่อง: Re: [MicrosoftDocs/edge-developer] Redesign TOC & Samples overview: Setup, GetStarts, SampApps, DeplSamps (PR #2054)


@nishitha-burman requested changes on this pull request.

________________________________

In microsoft-edge/webview2/code-samples-links.md<https://github.com/MicrosoftDocs/edge-developer/pull/2054#discussion_r922547418>:

>
-This article is about understanding what samples, `.sln` files, and `README.md` files are in the [WebView2Samples repo](https://github.com/MicrosoftEdge/WebView2Samples).
+Sample apps that use WebView2 are available in the **WebView2Samples** repo, for various frameworks or platforms.


We should add a link to the sample apps repo here: https://github.com/MicrosoftEdge/WebView2Samples/tree/main/SampleApps

________________________________

In microsoft-edge/webview2/code-samples-links.md<https://github.com/MicrosoftDocs/edge-developer/pull/2054#discussion_r922549181>:

>
-This particular `.sln` file doesn't include the Get Started projects, which are separate `.sln` files, one per platform.
+For some frameworks or platforms, the samples repo contains both a completed Getting Started tutorial and a sample app.  The sample apps demonstrate more features than the completed Getting Started tutorials.


The sample apps are home to sample code in the reference docs and conceptual docs. These apps showcase more of our APIs than then Getting Started tutorials.

________________________________

In microsoft-edge/webview2/code-samples-links.md<https://github.com/MicrosoftDocs/edge-developer/pull/2054#discussion_r922549986>:

> -
-The `WebView2Samples` repo contains the following projects.  Some of these projects are included in the `WebView2Samples.sln` file, but each platform also has a dedicated `.sln` file.
-
-#### Regular samples
-
-| Type of sample | Sample Project | Description |
-|---|---|---|
-| WinUI 2 (UWP) browser | [webview2_sample_uwp](https://github.com/MicrosoftEdge/WebView2Samples/blob/main/SampleApps/webview2_sample_uwp#readme) | Embeds a WebView2 control within a UWP application.<br/><br/>  Built as a UWP Visual Studio 2019 project.  Uses C++ and HTML/CSS/JavaScript in the WebView2 environment. |
-| Win32 C++ | [WebView2APISample](https://github.com/MicrosoftEdge/WebView2Samples/tree/main/SampleApps/WebView2APISample#readme) | Embeds a WebView2 control within a Win32 native application.  Illustrates a selection of WebView2 event handlers and API methods that allow a native Win32 application to directly interact with a WebView2 control and vice versa.<br/><br/>  Built as a Win32 project in [Visual Studio 2019](https://visualstudio.microsoft.com/vs/).  Uses C++ and HTML/CSS/JavaScript in the WebView2 environment.<br/><br/>The `.sln` file is in the parent, [SampleApps](https://github.com/MicrosoftEdge/WebView2Samples/tree/main/SampleApps) directory. |
-| Win32 C++ with Visual Composition | [WebView2SampleWinComp](https://github.com/MicrosoftEdge/WebView2Samples/tree/main/SampleApps/WebView2SampleWinComp) | Embeds a WebView2 control within a Win32 native application.  Uses [Windows Runtime Composition APIs](/uwp/api/windows.ui.composition), also called the Visual layer, to take advantage of the latest Windows 10 or later UI features and create better look, feel, and functionality in C++ Win32 applications.<br/><br/>  Built as a Win32 project in Visual Studio 2019.  Uses C++ and HTML/CSS/JavaScript in the WebView2 environment. |
-| WinForms | [WebView2WindowsFormsBrowser](https://github.com/MicrosoftEdge/WebView2Samples/tree/main/SampleApps/WebView2WindowsFormsBrowser) | Embeds a WebView2 control within a Windows Forms application.<br/><br/>  Built as a Windows Forms project in Visual Studio 2019.  Uses C# and HTML/CSS/JavaScript in the WebView2 environment. |
-| Windows Presentation Foundation (WPF) .NET | [WebView2WpfBrowser](https://github.com/MicrosoftEdge/WebView2Samples/tree/main/SampleApps/WebView2WpfBrowser) | Embeds a WebView2 control within a WPF application.<br/><br/>  Built as a WPF project in Visual Studio 2019.  Uses C# and HTML/CSS/JavaScript in the WebView2 environment. |
-| Chrome DevTools Protocol (CDP) in WPF | [WV2CDPExtensionWPFSample](https://github.com/MicrosoftEdge/WebView2Samples/tree/main/SampleApps/WV2CDPExtensionWPFSample) | Uses Chrome [DevTools Protocol](../devtools-protocol-chromium/index.md) functions using a `DevToolsProtocolHelper` object in WebView2.  Demonstrates the usage patterns of the WebView2 CDP extension in WPF.  This application is built with the [WebView2 CDP Extension](https://aka.ms/webviewcdp) that defines all CDP methods, events, and types.<br/><br/>  Built as a WPF project in Visual Studio 2019.  Uses C# in the WebView2 environment. |
-
-#### Deployment samples
+#### Articles, readmes, sample directories, and solution files


What is this section trying to convey?

**Review by JasonAndrewWriter — APPROVED** [review]

File: microsoft-edge/webview2/code-samples-links.md (hunk 3 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/code-samples-links.md:33]: removed
